src/x11/x.h

```diff
@@ -23,13 +23,15 @@ typedef struct _keyboard_shortcut {
         bool is_valid;
 } keyboard_shortcut;
 
+// Cyclical dependency
```

Comment:
For this specific case, the way to handle this would be to move `keyboard_shortcut` to `settings`, but that will have to wait for the next PR when X11 is fully decoupled from the rest of the code to allow for other outputs.